tests/jenkins/TestReleaseNotesCheck.groovy

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+import jenkins.tests.BuildPipelineTest
+import org.junit.Before
+import org.junit.Test
+import static com.lesfurets.jenkins.unit.global.lib.LibraryConfiguration.library
+import static com.lesfurets.jenkins.unit.global.lib.GitSource.gitSource
+
+class TestReleaseNotesCheck extends BuildPipelineTest {
+
+    String gitLogDate = '2022-10-10'
+    String comment = 'NO_COMMENT'
+    String gitIssueNumber = '123456'
+    String commentUniqueID = '123456'
+    String inputManifest = '2.2.0/opensearch-2.2.0.yml'
+
+    @Override
+    @Before
+    void setUp() {
+        super.setUp()
+
+        binding.setVariable('INPUT_MANIFEST', inputManifest)
+        binding.setVariable('GIT_LOG_DATE', gitLogDate)
+        binding.setVariable('COMMENT', comment)
+        binding.setVariable('GIT_ISSUE_NUMBER', gitIssueNumber)
+        binding.setVariable('COMMENT_UNIQUE_ID', commentUniqueID)
+        binding.setVariable('AGENT_X64','Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host')
+        binding.setVariable('dockerAgent', [image:'opensearchstaging/ci-runner:ci-runner-centos7-v1', args:'-e JAVA_HOME=/opt/java/openjdk-11'])
+
+    }
+
+    @Test
+    public void testReleaseNoteCheckPipeline() {
+        super.testPipeline("jenkins/release-notes-check/release-notes-check.jenkinsfile",
+                "tests/jenkins/jenkinsjob-regression-files/release-notes-check/release-notes-check.jenkinsfile")
+    }
```

Comment:
I believe we would like to assert few things like if the intended shell script has the expected calls. In future if the library makes some changes, we would like to catch that in our tests.

Reply:
@gaiksaya , makes sense.. I was thinking that as well, but debating with myself that the regression txt has the shared lib covered as well.. 
I also think it is better we do that you said .. I will add test to check if certain script/lib is called.. 
thanks!!